File: DocProxy/docproxy/src/main/java/org/ls/docproxy/IInvestor.java
package org.ls.docproxy;

/**
 * Author by Less on 2019/6/9.
 */
public interface IInvestor {

  /**
   * 登录股票账户
   * @param user
   * @param password
   */
  void login(String user, String password);

  /**
   * 买股票
   */
  void buyStock();

  /**
   * 卖股票
   */
  void sellStock();
}

File: DocProxy/app/src/main/java/org/ls/docproxy/proxy/IInvestor.kt
package org.ls.docproxy.proxy

/**
 * 先定义一个投资者接口
 */
interface IInvestor {

  /**
   * 登录股票账户
   * @param user
   * @param password
   */
  fun login(
    user: String,
    password: String
  )

  /**
   * 买股票
   */
  fun buyStock()

  /**
   * 卖股票
   */
  fun sellStock()

}